x-pack/legacy/plugins/epm/public/screens/add_data_source/step_one.tsx

```diff
@@ -3,19 +3,32 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-
+import React, { Fragment } from 'react';
 import {
   EuiDescribedFormGroup,
   EuiFieldText,
   EuiForm,
   EuiFormRow,
   EuiHorizontalRule,
-  EuiSpacer,
-  EuiSwitch,
+  EuiCheckboxGroup,
+  EuiCheckboxGroupOption,
 } from '@elastic/eui';
-import React, { Fragment } from 'react';
+import { FormState } from './add_data_source_form';
 
-export const StepOneTemplate = () => {
+interface AddDataSourceFormProps {
+  formState: FormState;
+  // EuiCheckboxGroup onChange prop type says parameter is an event, but it is a string of the input name
+  onCheckboxChange: (name: any) => void;
```

Comment:
I had a comment about the mismatch between the type & the comment, but I saw your convo w/the EUI team and it seems like this signature can change when we merge `master` into `feature-ingest`